src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final String DIVIDER = ",";
+    public static final String C_R_L_F = System.lineSeparator();
+    public static final int POS_OF_AMOUNT = 1;
+    public static final int POS_OF_NAME = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
You should have separate methods for reading data, calculating report and writing data and calling them here